.github/workflows/build-and-test.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - name: Mypy testing
         run: |
           # Not an exact mypy version, as we need 0.942 for pypy-3.8 support, but it's not available on 3.5
```

Comment:
"but it's not available on 3.5" also sounds like something that can be fixed now, not necessarily here

Reply:
Yes ... but I'm really good at managing the Debian ecosystem and releases; not so much the native Python ecosystem (pip, tox etc...)